### ISSUE TITLE
is there a script to run experiment with dice for comparison

### ISSUE THREAD

**Issue by Andrewtor** [jonasrothfuss/ProMP#3]
Hi, is there a script to run experiments with dice for comparing with other algorithms? I didn't see a script to do this under `run_script` directory. Thank you

**Comment by jonasrothfuss** [comment]
You can find execution scripts for DiCE in the run_script directory of the full_code branch.

**Comment by jonasrothfuss** [comment]
Be aware, that the naming of the classes an modules are slightly different in the full_code branch.

**Comment by Andrewtor** [comment]
So I checked full_code branch, and I run TRPO+DICE in ProMP/run_scripts/trpo_dice_run.py to compare with TRPO in ProMP/run_scripts/trpo_run.py, the result is attached, where blue line is TRPO while the other is DICE+TRPO, it clearly shows that DICE+TRPO here doesn't match with its result in paper. 
<img width="916" alt="results" src="https://user-images.githubusercontent.com/24619325/52748329-1e854e80-2f9b-11e9-9165-edb80d04c66a.png">

**Comment by Andrewtor** [comment]
@jonasrothfuss , could you have a look at this issue?

**Comment by jonasrothfuss** [comment]
Thanks for letting us know. Yes, we are aware of the problem. Results looked the same for us. We only got DiCE to work with vanilla policy gradients (VPG). That's why we make a VPG based comparison in Figure 3 of the paper. We spent a considerable amount of time, trying to make DICE+TRPO work but haven't managed to succeed. Due to time constraints, we stopped trying at some point.

I'd highly appreciate, if you take a closer look at the code and try to understand why DICE+TRPO doesn't learn anything. Maybe you are able to figure it out and make it work. Most likely there is just a stupid bug somewhere. If you have any questions, let me know.
